shim/build.rs

```diff
@@ -1,6 +1,58 @@
+use std::{env, ffi::OsStr, process::Command, path::PathBuf};
+
+
 fn main() {
-    println!("cargo:rustc-link-search=../cli");
+    let is_release = match env::var("PROFILE") {
```

Comment:
clippy complains that this could be a `matches` macro:

```
    let is_release = matches!(env::var("PROFILE"), Ok(profile) if profile == "release")
```

(Can't offer a suggested change because it's multiline)